lib/modules/manager/pep621/processors/pdm.ts

```diff
@@ -126,3 +127,44 @@ export class PdmProcessor implements PyProjectProcessor {
     }
   }
 }
+
+function generateCMDs(updatedDeps: Upgrade[]): string[] {
+  const cmds: string[] = [];
+  const packagesByCMD: Record<string, string[]> = {};
+  for (const dep of updatedDeps) {
+    switch (dep.depType) {
+      case depTypes.optionalDependencies: {
+        const [group, name] = dep.depName!.split('/');
```

Comment:
Maybe this could be replaces with `depGroup`, but I'm not sure there is some effort to use it outside of Gradle.